### DISCUSSION TITLE
VertexAI support

### DISCUSSION THREAD

**Discussion by eles13** [langfuse/langfuse#1627] [Ideas]
### Describe the feature or potential improvement

I am using [VertexAI model](https://github.com/langchain-ai/langchain-google/blob/main/libs/vertexai/langchain_google_vertexai/llms.py#L110) in instruct style, not ChatVertexAI, however, there is only model_name parsing for ChatVertexAI. I use LangchainCallbackHandler from your pypi package, there is no name parsing that I expect. Is there any way to add a few lines of code?

**Comment by marcklingen** [comment]
Thanks for raising this, open for PR, see python repo for model name parsing here: https://github.com/langfuse/langfuse-python/blob/54baa9b3ee6065c23c72926e5b3551051348433f/langfuse/extract_model.py#L40

We'll try to address this issue in a more generic way soon. The current implementation solves lot's of edge cases but also needs a lot of maintenance

**Comment by marcklingen** [comment]
This is fixed
